### PR TITLE
Change config semantics environments -> env

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

changed config keys
`environments` -> `envs`
`active` -> `current-env`

## Why?

shorter usage `tctl config set env.local.namespace` vs  `tctl config set environments.local.namespace`

`tctl config use-env` vs `tctl config use-environment`

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Comment by bergundy** [comment]
Maybe env should be a top level command.

`tctl env use ..`
`tctl env current`

File: pkg/config/defs.go [hunk 1]
```diff
@@ -23,7 +23,7 @@
 package config
 
 const (
-	KeyActive      string = "active"
-	KeyAlias       string = "aliases"
-	KeyEnvironment string = "environments"
+	KeyAliases            string = "aliases"
+	KeyCurrentEnvironment string = "current-env"
+	KeyEnvironments       string = "envs"
```
Comment (bergundy) [inline pkg/config/defs.go:28]: Maybe call this `env`?

Comment (feedmeapples) [reply pkg/config/defs.go:28]: since this lists a number of environments and not just a single env with vars, plural should be good?

Comment (feedmeapples) [reply pkg/config/defs.go:28]:
```
current-env: local
envs:
  local:
    namespace: default
  canary:
    namespace: canary
```

Comment (bergundy) [reply pkg/config/defs.go:28]: I'd expect:
```yml
current-env: local
env: # <-- singular
  local:
    namespace: default
  canary:
    namespace: canary
```

Then to use it'd be `tctl config set env.local.namespace default`

Comment (feedmeapples) [reply pkg/config/defs.go:28]: sure sg

**Review by bergundy — APPROVED** [review]

**Pull request merged**